srcpkgs/nagios/template

```diff
@@ -0,0 +1,57 @@
+# Template file for 'nagios'
+pkgname=nagios
+version=4.4.6
+revision=1
+build_style=gnu-configure
+
+_nagios_user="_nagios"
+_nagios_group="_nagios"
+_instdir="/usr/share/nagios"
+_bindir="/usr/bin"
+_vardir="/var/nagios"
+_confdir="/etc/nagios"
+_httpdconfdir="/etc/webapps/nagios/"
+_checkresultdir="/var/nagios/spool/checkresults"
+_cgibindir="/usr/libexec/nagios/cgi"
+
+configure_args="
+ --with-nagios-user=$_nagios_user
+ --with-nagios-group=$_nagios_group
+ --prefix=$_instdir
+ --bindir=$_bindir
+ --localstatedir=$_vardir
+ --sysconfdir=$_confdir
+ --with-httpd-conf=$_httpdconfdir
+ --with-checkresultdir=$_checkresultdir
+ --with-cgibindir=$_cgibindir
+ --enable-embedded-perl"
+make_build_args="all"
+make_install_args="install install-config"
+
+hostmakedepends="perl unzip"
+make_dirs="
+$_confdir 0775 _nagios _nagios
+$_vardir 0755 _nagios _nagios
+$_vardir/rw 0755 _nagios _nagios
+$_vardir/spool 0755 _nagios _nagios
+$_vardir/spool/checkresults 0755 _nagios _nagios"
+conf_files="/etc/nagios/nagios.cfg"
+
+short_desc="Nagios Core monitoring and alerting engine"
+maintainer="Dave Eddy <dave@daveeddy.com>"
+license="GPL-2.0-or-later"
+
+homepage="https://github.com/NagiosEnterprises/nagioscore"
+changelog="${homepage}/blob/master/Changelog"
+distfiles="${homepage}/releases/download/${pkgname}-${version}/${pkgname}-${version}.tar.gz"
+
+checksum=ab0d5a52caf01e6f4dcd84252c4eb5df5a24f90bb7f951f03875eef54f5ab0f4
+system_accounts="${_nagios_user}"
+
+post_install() {
+	# nagios has these set as 770 by default
+	chmod 755 "$DESTDIR/$_bindir/nagios"
+	chmod 755 "$DESTDIR/$_bindir/nagiostats"
```

Comment:
If they set it to `770`, they should have a reason. Allow your user by adding to `_nagios` group is better.

Reply:
Understood. will update to remove these commands.

Reply:
These files are owned by `root:root` by default, so only the root user or anyone in the root group will be able to execute `nagios` or `nagiostats`... this has issues using `chpst` to run the `nagios` executable (i'll detail more in the `chpst` comment).